client/components/connectionStatus/ConnectionStatusBar.stories.js

```diff
@@ -21,7 +21,7 @@ const stateDecorator =
 		);
 
 export default {
-	title: 'components/connectionStatus/ConnectionStatusBar',
+	title: 'components/ConnectionStatus',
```

Comment:
I believe the intent here was let the story title mimic the module file path.

Reply:
I think the best solution for this is `components/ConnectionStatusBar`. What do you think about it? 
Because it is redundant name when we see that in the folder "connectionStatus" we have just one component.